src/plugins/empty-org-recycle-bin/index.ts

```diff
@@ -0,0 +1,42 @@
+import { BrowserforcePlugin } from '../../plugin';
+
+const DELETE_EVENT_OBJECT = 'DeleteEvent';
+const LIST_VIEW_ALL_DELETE_EVENTS = 'AllDeleteEvents';
+const DELETE_EVENT_EMPTY_ALL_ACTION = 'DeleteEventEmptyAllAction';
+
+const PATHS = {
+  BASE: 'lightning/o/DeleteEvent/home'
+};
+
+const SELECTORS = {
+  EMPTY_BUTTON: '.modal-container .modal-footer button.uiButton:last-of-type'
+};
+
+export class EmptyOrgRecycleBin extends BrowserforcePlugin {
+  public async retrieve(definition?: Config): Promise<Config> {
+    return { enabled: false };
+  }
+
+  public async apply(config: Config): Promise<void> {
+    const deleteEventEmptyAllButtonSelector = `a.forceActionLink[title="${await this.determineDeleteEventEmptyAllActionLabel()}"]`;
+
+    const page = await this.browserforce.openPage(PATHS.BASE);
+    await page.waitForSelector(deleteEventEmptyAllButtonSelector);
+    await page.click(deleteEventEmptyAllButtonSelector);
+    await page.waitForSelector(SELECTORS.EMPTY_BUTTON);
+    await page.click(SELECTORS.EMPTY_BUTTON);
```

Comment:
In my tests, the recycle bin was not emptied.

Maybe we need to wait for a successful response from the server like this:

```suggestion
    await Promise.all([
      page.waitForResponse(
        response =>
          response
            .url()
            .includes(
              'RecycleBinActions.emptyOrgRecycleBin=1'
            ) && response.status() === 200
      ),
      page.click(SELECTORS.EMPTY_BUTTON)
    ]);
```